cli/src/config-schema.json

```diff
@@ -792,6 +792,17 @@
                 }
             }
         },
+        "trailer": {
+            "type": "object",
+            "description": "Trailer configuration",
+            "properties": {
+                "separators": {
+                    "type": "string",
+                    "description": "A set of characters recognized as trailer separators",
```

Comment:
Is it still true that the first of these is used by `jj` by default, like [git](https://git-scm.com/docs/git-interpret-trailers#Documentation/git-interpret-trailers.txt-trailerseparators)? If so, I'd mention it here.

Reply:
We copy the trailers as generated by the `commit_trailers` template configured by the user, so we don't use the first as the default value.
I suppose it could be done in `format_change_id_trailer()`, but for now `:` is used.